packages/devtools-source-map/RELEASE.md

```diff
@@ -0,0 +1,34 @@
+### How to create a new devtools-source-map release for mozilla-central.
+
+Before you get started, you will need clones of devtools-source-map and
+mozilla-central, as well as a Bugzilla account.
+
+```
+1. Prepare devtools-source-map release
+  a. create a release branch on github
+  b. update minor version in package.json, reset build version:
+    v0.N.X -> v0.N+1.0
+  c. commit `release: bump version to v0.N+1.0`
+  d. create a PR on Github for this release
+  e. make sure you have a configs/local.json file, with `firefox.mcPath` pointing to your mozilla-central clone
+
+2. Copy to mozilla central
+  a. update your mozilla-central clone to the latest
+  b. in devtools-source-map, run node bin/copy-assets.js`
+  c. create a new bug in Developer Tools [1]
+  d. commit 'Bug XXXXXXX - devtools-source-map v0.N+1.0: update devtools-source-map bundle from GitHub;r=reviewer'
+
+3. Validate & cleanup
+  a. push to try, test locally, submit for review etc ...
+  b. while try fails or some problem is detected, go back to devtools-source-map, fix the issue,
+     create a new bundle and go back to 2.a
+  c. when everything is fine and the patch is r+, land on autoland/inbound
+  d. merge the PR on github
+  e. create a tag for v0.N+1.0 on github
```

Comment:
That implies that every commit would need to apply to only a single package, doesn't it? It seems to negate some of the benefits of working in a mono-multi repo, since there will be cases where you'd want to make changes across multiple packages at once. Babel's repo, for example, seems to allow multi-package changes in a single commit. So, we should think about what we want to allow for ourselves.

If we're trying to grep changes for release notes, maybe that is better solved with tooling that filters commits based on file paths changed?